packages/0x.js/src/fetchers/simple_order_filled_cancelled_fetcher.ts

```diff
@@ -0,0 +1,24 @@
+import { BlockParamLiteral } from '@0xproject/types';
+import { BigNumber } from '@0xproject/utils';
+
+import {OrderFilledCancelledFetcher} from '../abstract/order_filled_cancelled_fetcher';
+import {ExchangeWrapper} from '../contract_wrappers/exchange_wrapper';
+
+export class SimpleOrderFilledCancelledFetcher implements OrderFilledCancelledFetcher {
+    private _exchange: ExchangeWrapper;
+    constructor(exchange: ExchangeWrapper) {
```

Comment:
Should this also accept a `BlockParamLiteral` like `BalanceAndProxyAllowanceFetcher `? It would require a refactor of `OrderFilledCancelledLazyStore`

Reply:
I'm ok with changing the simple fetchers to accept state layer, should I also modify getOrderState with a second optional param for state layer?

Reply:
I assumed the answer is yes and went ahead and modified getOrderState to support a state param.